cc-pk-service/src/handleData.ts

```diff
@@ -160,12 +166,12 @@ export async function getReusables(userSurveys:Survey[]) {
             }
             items.push(item);
             }
+        } catch (err) {
+            // Not really an error situation, this is just used to inform which survey is missing data
+            console.log(`Survey with id ${userSurveys[i].id} is missing building information`);

```

Comment:
Tested this and now other surveys get through. Now we maybe should think of a way to transport this information to backend and ui. But very good!